e2e-tests/cypress/e2e/journeys/quote/tell-us-about-your-policy/tell-us-about-your-single-policy-validation.spec.js

```diff
@@ -153,4 +153,16 @@ context('Tell us about the policy you need page - form validation', () => {
         .should('have.attr', 'value', '10');
     });
   });
+
+  describe('with the `perecntage of cover` field has been submitted and other fields are not provided', () => {
+    it('should render the submitted values for the `percentage of cover` field', () => {
+      tellUsAboutYourPolicyPage[FIELD_IDS.CONTRACT_VALUE].input().clear();
+
+      tellUsAboutYourPolicyPage[FIELD_IDS.PERCENTAGE_OF_COVER].input().select('90');
```

Comment:
not a blocker.  Can this be const ___ = 90 and same for above, so we only change it in one place if needed